Filter/ClassFilter.php

```diff
@@ -66,13 +66,24 @@ public function getFieldType()
      */
     public function getFieldOptions()
     {
-        return $this->getOption('choices', array(
+        if (!class_exists('Symfony\Component\Form\Extension\Core\ChoiceList\ChoiceList')) {
+            $choiceOption = array(
+                'choices' => $this->getOption('sub_classes'),
+                'choices_as_values' => true, //NEXT_MAJOR: Remove this line when drop Symfony <3.0 support
+            );
+        } else {
+            //NEXT_MAJOR: Remove this condition when drop Symfony <2.7 support
```

Comment:
We have discussion about this comment in [#691](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/issues/691#issuecomment-311638009). The goal of the comments is to make a note why this needed. You are want that I dropping this comment?